### DISCUSSION TITLE
TLS Fingerprint?

### DISCUSSION THREAD

**Discussion by volker001** [kkapsner/CanvasBlocker#604] [General]
Is a TLS Fingerprint undesirable?
https://browserleaks.com/ssl
https://ja3er.com/

**Comment by kkapsner** [comment]
I haven't looked into that in detail. A WebExtension is not able to change the TLS behaviour of the browser/operating system.

But I think this fingerprint will be shared with a lot of other people. So as long as you do not a very rare combo (e.g a very old browser) and did not tinker with the TLS settings you should be fairly protected from tracking.
